app/src/main/java/org/stepik/android/domain/analytic/interactor/AnalyticInteractor.kt

```diff
@@ -31,8 +30,8 @@ constructor(
                 bundle.keySet()?.forEach {
                     properties.add(it, gson.toJsonTree(bundle[it]))
                 }
-                (properties[PARAM_DATA] as? JsonObject)
-                    ?.add(PARAM_LANGUAGE, JsonPrimitive(sharedPreferencesHelper.languageForFeatured))
+//                (properties[PARAM_DATA] as? JsonObject)
+//                    ?.add(PARAM_LANGUAGE, JsonPrimitive(sharedPreferencesHelper.languageForFeatured))
```

Comment:
lang param should remain for other types of events
for query events it should sync with one from Filter